### PR TITLE
feat: Make badge preview to support chain agnostic urls

### PR DESCRIPTION
**Pull request by Agupane**
# Description
Depends on #276 

- The badge preview now support chain-agnostics urls so it will render badges from all the available networks and deployed contracts
- Note: the changes on subgraph-endpoints-dev.json should be reverted before merging this pr

### CODE REVIEW

File: src/components/header/Header.tsx [hunk 1]
```diff
@@ -32,14 +30,9 @@ const HeaderContainer = styled(Box)(({ theme }) => ({
 }))
 
 const Header = () => {
-  const isMobile = useSizeSM()
-  const updateAccountCenter = useAccountCenter()
-  const { connectWallet, isWalletConnected } = useWeb3Connection()
   const { t } = useTranslation()
-
-  useEffect(() => {
-    updateAccountCenter({ enabled: isMobile })
```
Comment (ramabit) [inline src/components/header/Header.tsx:41]: is this update not necessary anymore? no idea what was that for

File: src/config/web3.ts [hunk 2]
```diff
@@ -111,13 +120,23 @@ export const chainsConfig: Record<ChainsValues, ChainConfig> = {
...
   //   name: 'Mainnet',
-  //   shortName: 'Mainnet',
+  //   shortName: ChainShortName.mainnet,
   //   chainId: Chains.mainnet,
   //   chainIdHex: '0x1',
   //   rpcUrl: getProviderUrl(Chains.mainnet),
   //   blockExplorerUrls: ['https://etherscan.io/'],
   //   token: 'ETH',
   // },
+  // [Chains.mainnet]: {
```
Comment (ramabit) [inline src/config/web3.ts:130]: Chains.polygon will be here

File: src/hooks/nextjs/useStatisticsVisibility.tsx [hunk 3]
```diff
@@ -68,7 +68,7 @@ export default function useStatisticVisibility(
 
 function getStoredStatisticVisibility(
   category: string,
-  address: string | null,
+  address: string | undefined,
```
Comment (ramabit) [inline src/hooks/nextjs/useStatisticsVisibility.tsx:71]: address?: string

**Comment by ramabit** [comment]
@Agupane please take a look at the comments and resolve the conflicts

**Review by FedeMadoery — APPROVED** [review]

**Pull request merged**